samples/async-messaging-patterns/README.md

```diff
@@ -0,0 +1,289 @@
+# Async Messaging Communication Patterns
+
+## Introduction
+
+There're several messaging domains available for asynchronous messaging
+
+**Message queues**
+
+Provides delivery guarantees and preserves the order of the messages. If several consumers are bound to a queue only one 
```

Comment:
can you recheck this sentence ?

Reply:
Fixed 83c6660fc9017b82e21b8a8c1f0e1eee53e8da8b